modules.d/95resume/module-setup.sh

```diff
@@ -2,6 +2,9 @@
 
 # called by dracut
 check() {
+    # shellcheck source=dracut-dev-lib.sh
+    . "${dracutsysrootdir}${dracutbasedir}"/modules.d/99base/dracut-dev-lib.sh
```

Comment:
Do we really need to move the boot-specific `getcmdline` function to `dracut-dev-lib.sh` and source it only for this? AFAIK `/etc/cmdline` and  `/etc/cmdline.d` are a dracut specific thing (ignored by systemd, and therefore by `systemd-hibernate-resume-generator`, see https://github.com/systemd/systemd/issues/22935) available only in the initrd (not on the host).

If the only concern of this PR is fixing the bug introduced by #2160, I'd just prefer @martinwhitaker's patch ( https://github.com/dracutdevs/dracut/pull/2160#issuecomment-1464496727), removing `/etc/cmdline` and  `/etc/cmdline.d` from the grep.

Reply:
Maybe @Henrik66 understands it, because he didn't ask. Or is it another PR thought up by you and submitted by him so you can count your approval for your own PR?

boot-specific => it's a function provided by `dracut-lib.sh`, script which is included and used in the initrd only (at boot) by the scripts called from there (e.g. hooks).